ui/components/component-library/tag-url/tag-url.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+import { Text } from '../text';
+import {
+  ALIGN_ITEMS,
+  BORDER_RADIUS,
+  COLORS,
+  DISPLAY,
+  SIZES,
+  TEXT,
+} from '../../../helpers/constants/design-system';
+import { AvatarFavicon } from '../avatar-favicon';
+import { ButtonLink } from '../button-link';
+import { Icon, ICON_NAMES } from '../icon';
+
+export const TagUrl = ({
+  label,
+  labelProps,
+  actionButtonLabel,
+  actionButtonProps,
+  src,
+  avatarFaviconProps,
+  fallbackIconProps,
+  className,
+  ...props
+}) => {
+  return (
+    <Box
+      className={classnames('mm-tag-url', className)}
+      backgroundColor={COLORS.BACKGROUND_DEFAULT}
+      borderColor={COLORS.BORDER_DEFAULT}
+      borderWidth={1}
+      alignItems={ALIGN_ITEMS.CENTER}
+      paddingLeft={2}
+      paddingRight={4}
+      gap={2}
+      borderRadius={BORDER_RADIUS.PILL}
+      display={DISPLAY.INLINE_FLEX}
+      {...props}
+    >
+      <AvatarFavicon imageSource={src} {...avatarFaviconProps} />
+      <Icon
+        className="tag-url__lock-icon"
+        name={ICON_NAMES.LOCK_FILLED}
+        color={COLORS.ICON_DEFAULT}
```

Comment:
I think the default color is `ICON_ALTERNATIVE` in the Figma file

<img width="658" alt="Screen Shot 2022-11-02 at 6 55 48 PM" src="https://user-images.githubusercontent.com/8112138/199629070-a9ff6717-7750-4cc6-9ff8-7d45fbb67281.png">

```suggestion
        color={COLORS.ICON_ALTERNATIVE}
```